### ISSUE TITLE
[BUG] __class__ not set defining 'SubClass' as <class '__main__.SubClass'>

### ISSUE THREAD

**Issue by AlecHaring** [dashingsoft/pyarmor#1062]
There seems to be an issue with subclasses. When running an obfuscated script that has a class that inherits another class, it produces the following error:
```
Traceback (most recent call last):
  File "<frozen __main__>", line 3, in <module>
  File "<frozen test>", line 9, in <module>
RuntimeError: __class__ not set defining 'SubClass' as <class '__main__.SubClass'>. Was __classcell__ propagated to type.__new__?
```

**Executed command:** `pyarmor -d gen .\pyarmor_test\test.py`

**test.py**
```python
class MainClass:
    def __init__(self):
        pass

    def test(self):
        print("MainClass.test")


class SubClass(MainClass):
    def __init__(self):
        super().__init__()

    def test(self):
        print("SubClass.test")


if __name__ == "__main__":
    MainClass().test()
    SubClass().test()
```

**Comment by jondy** [comment]
Fixed in dev version `8.1.dev1`

**Comment by AlecHaring** [comment]
Confirmed, fixed :)
